src/test/java/io/strimzi/kafka/bridge/http/ProducerIT.java

```diff
@@ -274,7 +275,7 @@ void sendTextMessageWithKey(VertxTestContext context) throws InterruptedExceptio
                 new ByteArrayDeserializer(), new ByteArrayDeserializer());
         consumer.handler(record -> {
             context.verify(() -> {
-                assertThat(new String(record.value()), is(value));
+                assertThat(new String(record.value()).replace("\\", ""), is(value));
```

Comment:
why?

Reply:
I was just checking the issue with backslashes :)

Reply:
@ppatierno is it okay if we use a decoder in tests in order to get rid of `\`?

Reply:
I don't see why you want to do this replace. There is no `\`.
The `text` converter is used and the test just works fine by asserting:

```java
assertThat(new String(record.value()), is(value));
```